src/services/donationService.ts

```diff
@@ -529,7 +529,7 @@ export async function getDonationToGivethWithDonationBoxMetrics(
     throw new Error('giveth project not found!');
   }
 
-  const { donationsToGiveth, pairedDonations } = await findRelevantDonations(
+  const donationsToGiveth = await findDonationsToGiveth(
     startDate,
     endDate,
     givethProject.id,
```

Comment:
`const givethProject = await findProjectIdBySlug('giveth');`
This line of code only fetches Giveth project in staging. In production the slug is different, also the slug maybe changed in future so it's better to fetch projects from DB by ID (not slug).
Giveth project ID is `1` in both staging and prod, and you don't need to fetch that from DB. You can consider that as a constant.

Reply:
If we know the given project ID is always 1, we can do this.

Reply:
Done